extensions/Worker.Extensions.ServiceBus/src/ServiceBusSessionMessageActionsConverter.cs

```diff
@@ -58,5 +54,32 @@ public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
                 return new ValueTask<ConversionResult>(ConversionResult.Failed(exception));
             }
         }
+
+        private string ParseSessionIdFromBindingData(ConverterContext context)
+        {
+            // Try to resolve sessionId directly
+            var bindingData = context.FunctionContext.BindingContext.BindingData;
+            bindingData.TryGetValue("SessionId", out object? sessionId);
```

Comment:
Can we make `SessionId` and `SessionIdArray` constants?

Reply:
done! made the other strings consts as well